enhancements/installer/image-based-installer.md

```diff
@@ -0,0 +1,331 @@
+
+N/A
+
+## Version Skew Strategy
+
+### Configuration ISO version and Seed OCI Image version
+
+The IBI configuration ISO content version should be compatible with the seed
+OCI image version contained in the IBI installation ISO. The Lifecycle Agent
+Operator, which is contained in the installation ISO, is responsible for
+ensuring the compatibility of those two at the time of the image-based
+installation. The configuration version does not depend on the OpenShift
+versions used to generate either one of the IBI artifacts (i.e. configuration
+ISO and installation ISO). In the case of incompatibility between the
+configuration version and the seed OCI image version, the image-based
+installation will fail with the respective error message.
+
+### Installation ISO RHCOS ISO version and Seed OCI Image version
+
+The RHCOS base ISO, which is contained in the IBI installation ISO and derived
+from the OpenShift release payload, has no strict requirements to be tied to the
+seed OCI image OpenShift version. The features and configuration of the
+underlying tools required to successfully complete an image-based
+installation, e.g. Podman, cri-o, SELinux, ostree, do not frequently change
+between OpenShift versions. The risk of version skew is small and in such case
+the image-based installation will fail with the respective error message.
+
+## Operational Aspects of API Extensions
+
+N/A
+
+## Support Procedures
+
+N/A
+
+## Alternatives
+
+### Downloading the installation ISO from Assisted Installer SaaS
+
+The Assisted Installer SaaS could be used to generate and serve the image-based
+installation ISO, which would potentially enhance the user experience, compared
+to configuring and executing a command-line tool. In addition, even in a fully
+disconnected environment, the user could generate the ISO via the Assisted
+Installer service and then carry it over to the disconnected environment.
+
+However, for the Assisted Installer service this would be a completely new
+scope, as the image-based installation ISO is not generated per cluster, but for
+multiple SNO clusters with similar underlying hardware (due to the requirements
+of the IBI flow) and its configuration input is different than what is currently
+supported.
+
+### Building the installation ISO in the Lifecycle Agent Operator
+
+The Lifecycle Agent Operator could be used to generate and serve the image-based
+installation ISO, as this is the component used to generate the seed OCI
+image, which is the basis of the IBI flow.
+
+However, the following reasons constitute a separate command-line tool a better
+fit:
+
+- the seed OCI image is not only used for IBI, but also for the
+  Image-based Upgrade (IBU) flow, supported by different user personas.
+- the same seed OCI image can be used to generate multiple image-based
+  installation ISOs (e.g. with different installation disks).
+- the user persona to generate the image-based installation ISO must have a
```

Comment:
I think this is the key point against this option, the seed generation is potentially performed by a different persona (release/certification/other team) than the team that creates the installation ISO and install the nodes at the factory.

Reply:
I'll move it as the first point and rewrite it a bit to reflect exactly what you explained :+1: